tests/BinEnvJsonTest.php

```
@@ -0,0 +1,339 @@
+    private function removeDirectory(string $dir): void
+    {
+        if (! is_dir($dir)) {
+            return;
+        }
+
+        $items = new RecursiveIteratorIterator(
+            new RecursiveDirectoryIterator($dir, RecursiveDirectoryIterator::SKIP_DOTS),
+            RecursiveIteratorIterator::CHILD_FIRST,
+        );
+        foreach ($items as $item) {
+            if ($item->isDir() && ! $item->isLink()) {
```

PHPStan (2.0.3)

Cannot call method isDir() on mixed.


(method.nonObject)